src/internal_events/aws_kinesis_firehose.rs

```diff
@@ -26,10 +26,9 @@ impl<'a> InternalEvent for AwsKinesisFirehoseRequestReceived<'a> {
 }
 
 #[derive(Debug)]
-pub struct AwsKinesisFirehoseRequestError<'a> {
-    pub request_id: Option<&'a str>,
```

Comment:
@jdrouet it appears this `request_id` is not actually used when emitting this error. Should the attribute be removed here or should it be added to the logging?

Reply:
good point, I'll add it to the log 😉

Reply:
@bruceg here it is https://github.com/vectordotdev/vector/pull/11383/commits/00e837b3dfe97096a60057c8b3f51e5976064d26